package.json

```diff
@@ -20,7 +20,7 @@
     "node": "^18.x || ^20.x"
   },
   "dependencies": {
-    "@aurodesignsystem/auro-dropdown": "^2.10.0",
+    "@aurodesignsystem/auro-dropdown": "^2.10.2",
```

Comment:
just leaving comment to remind that this will need to be updated again to latest version after https://github.com/AlaskaAirlines/auro-dropdown/pull/217 is merged & released.